### ISSUE TITLE
Can't delete values with `updateEmptyPropertiesOnly`

### ISSUE THREAD

**Issue by signalwerk** [punktDe/nodereplicator#14]
Imagine I have in two dimensions content:

## Dimension DE
title: "Title DE"

## Dimension FR
title: "Title FR"

I have `updateEmptyPropertiesOnly` set to `true`

```yaml
  options:
    replication:
      structure: true
      content: true
      updateEmptyPropertiesOnly: true
      createHidden: true
```

Now I would like to delete the content of the field in both dimensions.

Since I only expect an update in the other dimension if the other dimension is empty (and not the current) I would think these steps should delete the content:

* go to DE-Dimension
* delete title (in FR-Dimension is still the text)

But now instead of 

## Dimension DE
title: ""

## Dimension FR
title: "Title FR"

I get

## Dimension DE
title: "Title FR"

## Dimension FR
title: "Title FR"

Basically, `updateEmptyPropertiesOnly` is not aware of my current dimension. I would assume the content I enter (even if it's an empty string) is the master and gets copied to the other dimension (or not if not empty) but with an empty value, it happens the other way around.

**Comment by daniellienert** [comment]
Hey @signalwerk, 

Thanks for the detailed issue description. I fear, that this is not how it is meant to work. If `updateEmptyPropertiesOnly`. is set, it only updates the target property if the **target property is empty**. The idea was that you do not want to change already adjusted properties in the target dimension

**Comment by daniellienert** [comment]
Clarified in https://github.com/punktDe/nodereplicator/pull/15/files

**Comment by signalwerk** [comment]
@daniellienert thank you for your response. I completely understand what you say, and it was actually what I was expecting. But you are talking about the target property. The problem is that it is basically not accepting the property in the current dimension. I change the current property and it's not stored. I guess this is not intuitive and not intended. 

Everything works as expected as long as I fill in a value that is not empty. As soon as I fill in an empty string, it does behave different in the current dimension and gets the value of the target dimension. sorry, very hard to explain. but my example above should explain it better.
